torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18782,24 +18920,79 @@ def __init__(
         ),
     ),
     #
-    # Reduction OpInfos
+    # Reduction Reference OpInfos
     #
     ReductionPythonRefInfo(
         "_refs.sum",
         torch_opinfo_name="sum",
         supports_out=True,
     ),
-
     ReductionPythonRefInfo(
         "_refs.amin",
         torch_opinfo_name="amin",
     ),
-
     ReductionPythonRefInfo(
         "_refs.amax",
         torch_opinfo_name="amax",
-    )
-
+    ),
+    #
+    # Tensor Creation Reference OpInfos
+    #
+    PythonRefInfo(
+        "_refs.empty",
+        torch_opinfo_name="empty",
+        skips=(
+            DecorateInfo(unittest.skip("Expected: empty is not comparable"),
+                         'TestCommon',
+                         'test_python_reference_consistency'),
+            DecorateInfo(unittest.skip("Expected: empty is not comparable"),
+                         'TestCommon',
+                         'test_out'),
+            DecorateInfo(unittest.skip("Expected: empty is not comparable"),
+                         'TestCommon',
+                         'test_out_warning'),
+            # Test doesn't support non-tensor inputs
+            DecorateInfo(unittest.expectedFailure,
+                         'TestMathBits',
+                         'test_conj_view'),
+            # Test doesn't support non-tensor inputs
+            DecorateInfo(unittest.expectedFailure,
+                         'TestMathBits',
+                         'test_neg_conj_view'),
+            # Test doesn't support non-tensor inputs
+            DecorateInfo(unittest.expectedFailure,
+                         'TestMathBits',
+                         'test_neg_view'),
```

Comment:
join @zou3519 and I in advocating these skips should be automatically generated and saved ;) It's very time consuming to manually track all of these down

Reply:
Yeah but not in this PR

Reply:
I think you may already be past the tipping point where it will be faster to sit down and add this infrastructure than to play popcorn with the CI for the next week